.husky/pre-commit

```diff
@@ -4,5 +4,6 @@
 npm run format:fix
 npm run lint:fix
 npm run typecheck
+npm run update-toc
```

Comment:
Will this only add the TOC to the root directory files? You have removed the .github files from the PR, but you still haven't answered my question.

Reply:
Yes, I am only adding a Table of Contents to the root directory. I haven't made any changes to the .github files, so I'm unsure why the .github folder would be included in the pull request. I apologize if I didn't understand your question correctly.

![toc](https://github.com/PalisadoesFoundation/talawa-admin/assets/66772290/a86fd4ed-40f0-4831-b135-1fb4fbbd82e8)